model.py

```diff
@@ -9066,19 +9094,6 @@ def works_updated_since(self, _db, timestamp):
      UniqueConstraint('collection_id', 'library_id'),
  )
 
-    
-collections_licensepools = Table(
-    'collections_licensepools', Base.metadata,
-     Column(
-         'collection_id', Integer, ForeignKey('collections.id'),
-         index=True, nullable=False
-     ),
-     Column(
-         'licensepool_id', Integer, ForeignKey('licensepools.id'),
-         index=True, nullable=False
-     ),
-     UniqueConstraint('collection_id', 'licensepool_id'),
-)
```

Comment:
This table isn't present in the master branch, so there are no real instances that use it. I created it thinking we could make collection-licensepool many-to-many, but that didn't work out.